src/UIBook/Widgets/Placeholder.elm

```diff
@@ -0,0 +1,122 @@
+module UIBook.Widgets.Placeholder exposing
+    ( placeholder
+    , custom, withBackgroundColor, withForegroundColor, withHeight, withWidth, view
+    )
+
+{-| An utility Widget that serves as a placeholder for the content of sections in your chapters
+
+You can create a Placeholder for each one of your sections
+
+    import UIBook.Widgets.Placeholder exposing (placeholder)
+
+    card : Html msg -> Html msg
+    card child = ...
+
+    chapter "Card"
+        |> withSection
+            ( card placeholder )
+
+@docs placeholder
+
+You can also customize several aspects of the placeholder widget, including height, width, background color and foreground color.
+
+    chapter "Placeholder"
+    |> withSections
+        [ ( "Custom"
+            , Placeholder.custom
+                |> Placeholder.withWidth 400.0
+                |> Placeholder.withForegroundColor "#FFFFFF"
+                |> Placeholder.view
+            )
+        ]
+
+@docs custom, withBackgroundColor, withForegroundColor, withHeight, withWidth, view
```

Comment:
It's usually a good practice to order these based on how useful they are.

Based on this, the "initializer" and "finalizer" should probably upfront. This way we make sure the user can already see the code example that will be on top of them.

After that, I'd guess the most useful settings would be sizes, then foreground, then background, so:

```suggestion
@docs custom, view, withHeight, withWidth, withForegroundColor, withBackgroundColor
```